src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/ClientBaseTests.cs

```diff
@@ -174,6 +175,163 @@ public static void ClientBaseOfT_Sync_RoundTrip_Call_Using_NetTcpTransport()
         }
     }
 
+    [Fact]
```

Comment:
Based on following the general same tree structure as is used by the published WCF Samples, the Streaming sample is located 'WF_WCF_Samples\WCF\Basic\Contract\Service'. Service in this case refers to "Service Contract". So we would need to add a new directory under 'System.Private.ServiceModel\tests\Scenarios\Contract'.